torch/_inductor/codegen/cpp_bmm_template.py

```diff
@@ -14,10 +14,13 @@
 from .cpp_utils import DTYPE_TO_CPP, GemmBlocking
 
 
+# We use BY as the output in order to cause sizevars in the batch dim to be set in the kernel
```

Comment:
But using `BY` as the output doesn't align with the semantics of these functions which expect "Y" is a slice of `BY` instead?